src/Expectation/Expectation.php

```diff
@@ -164,7 +164,7 @@ public function __call($name, array $arguments = [])
     }
 
     /**
-     * @return \Mockery\Expectation|\Mockery\CompositeExpectation
+     * @return \Mockery\Expectation|\Mockery\ExpectationInterface
```

Comment:
This is "correct", however, because of how \Mockery is written `ExpectationInterface` is missing many methods that `CompositeExpectation` has. And I can't use `CompositeExpectation` in the constructor, because `CompositeExpectation` is created in a "fluent" interface.
This means that if I use `ExpectationInterface` the IDE will not suggest a lot of methods that are actually there.

These are the consequences of using fluent methods and magic methods. In this case I have to choose between a correct doc bloc and poor dev experience (no autocomplete) and wrong doc bloc and better dev experience... and honestly I prefer the latest.